### DISCUSSION TITLE
Smooth plastic material (plastic) with texture

### DISCUSSION THREAD

**Discussion by Gzhji** [mitsuba-renderer/mitsuba3#1142] [Q&A]
Dear community,

I am trying to set up [smooth plastic material (plastic)](https://mitsuba.readthedocs.io/en/latest/src/generated/plugins_bsdfs.html#blended-material-blendbsdf) with texture as in the following image. Does anyone know how to edit the code snippet for this? 

Thanks in advance!
Guanzhou

![image](https://github.com/mitsuba-renderer/mitsuba3/assets/70720549/ffbacdd7-56b0-4959-998e-0b87060a3e57)

**Comment by njroussel** [comment]
Hi @Gzhji

You have to edit the `diffuse_reflectance` parameter to be a texture rather than an rgb color.
For example:
```py
'type': 'plastic',
'diffuse_reflectance': {
    'type': 'bitmap',
    'filename': 'your/path/to/file.png'
},
'int_ior': 1.9
```
